<ui version="4.0" >
 <class>Form</class>
 <widget class="QWidget" name="Form" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>546</width>
    <height>546</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item>
    <widget class="QGroupBox" name="gradientOptions" >
     <property name="title" >
      <string/>
     </property>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QGroupBox" name="gradientColorsBox" >
        <property name="title" >
         <string>Colors</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="gradientTypeBox" >
        <property name="title" >
         <string>Type</string>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QRadioButton" name="linearTypeButton" >
           <property name="text" >
            <string>Linear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radialTypeButton" >
           <property name="text" >
            <string>Radial</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="conicalTypeButton" >
           <property name="text" >
            <string>Conical</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="spreadMethodBox" >
        <property name="title" >
         <string>Spread method</string>
        </property>
        <layout class="QVBoxLayout" >
         <property name="margin" >
          <number>9</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QRadioButton" name="padSpreadButton" >
           <property name="text" >
            <string>Pad</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="repeatSpreadButton" >
           <property name="text" >
            <string>Repeat</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="reflectSpreadButton" >
           <property name="text" >
            <string>Reflect</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
